modules/ROOT/pages/dw-core-functions-contains.adoc

```diff
@@ -3,27 +3,39 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-//* <<contains1>>
-//* <<contains2>>
-//* <<contains3>>
-
 
 [[contains1]]
 == contains(Array<T>, Any): Boolean
 
-Indicates whether an array contains a given value. Returns `true` or `false`.
+Returns `true` if a list (array) contains a given value, `false` if not.
+
+
+=== Parameters
+
+[%header, cols="1,3"]
+|===
+| Name   | Description
+| `items` | The input list (an `Array`).
```

Comment:
Array of values (any restrictions on the value types? i.e. can they also be any supported data type?)